start.d.ts

```diff
@@ -0,0 +1,2 @@
+import agent from './';
```

Comment:
you can avoid creating this file by doing 
```ts
declare module 'elastic-apm-node/start' {
   export * from './'
}
```

Reply:
I'm getting this error in VS Code if I do:

![image](https://user-images.githubusercontent.com/10602/54674430-b3301e80-4afc-11e9-8355-b706b61bb0ff.png)

Reply:
sorry my bad, we can do this `export default Agent`

Reply:
That would export the `Agent` object though which we're not doing in real life. We're exporting an instance of the `Agent` object